src/Sylius/Component/Review/Model/ReviewableInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Review\Model;
+
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface ReviewableInterface
+{
+    /**
+     * @return string
+     */
+    public function getName();
+
+    /**
+     * @return Collection|ReviewInterface[]
```

Comment:
If we remove Collection from typehint we can decouple this interface from doctrine commons